Externals/curl/lib/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 if(NOT CMAKE_SYSTEM_NAME MATCHES "Windows")
```

Comment:
Given, this now has an else case, it'd likely be better to invert this as:

```cmake
if(CMAKE_SYSTEM_NAME MATCHES "Windows")
    add_definitions(-DUSE_SCHANNEL -DUSE_WINDOWS_SSPI)
else()
    add_definitions(-DHAVE_CONFIG_H)
endif()
```

so that the `NOT` can be dropped, making for less reading.